example/client/KatranSimpleClient.cpp

```diff
@@ -114,12 +119,32 @@ void KatranSimpleClient::delService(const std::string &address, int proto) {
   }
 }
 
+void KatranSimpleClient::updateReal(const std::string &address, uint32_t flags, bool setFlags) {
+  LOG(INFO) << folly::sformat("Updating real: {} {}", address, proto);
+  RealMeta realMeta;
+  realMeta.address = address;
+  realMeta.flags = flags;
+  realMeta.setFlags = setFlags;
+
+  if (client_->sync_modifyReal(std::move(realMeta))) {
+    LOG(INFO) << "Real updated";
+  } else {
+    LOG(ERROR) << "ERROR: Real not updated";
+  }
+}
+
 void KatranSimpleClient::updateServerForVip(const std::string &vipAddr,
                                             int proto,
                                             const std::string &realAddr,
-                                            uint64_t weight, bool del) {
+                                            uint64_t weight,
+                                            const std::string &flags,
+                                            bool del) {
   auto vip = parseToVip(vipAddr, proto);
-  auto real = parseToReal(realAddr, weight);
+  if (it == realFlagTranslationTable.cend()) {
```

Comment:
You missed declaration of it here.
  `const auto& it = realFlagTranslationTable.find(flags);`